invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/util.js

```diff
@@ -15,14 +15,22 @@ import _camelCase from "lodash/camelCase";
 export function createSearchAppInit(
   defaultComponents,
   autoInit = true,
-  autoInitDataAttr = "invenio-search-config"
+  autoInitDataAttr = "invenio-search-config",
+  allowMultipleApps = false
 ) {
   const initSearchApp = (rootElement) => {
-    const config = JSON.parse(
+    const { appId, ...config } = JSON.parse(
```

Comment:
It would be safer to add a default value here (even if it was not there before).

Reply:
actually intentionally I didn't, because if not passed or is a falsy value then the namespace will not kick in and existing instances that have overridden components will not break...